packages/perseus/src/types.ts

```diff
@@ -141,6 +142,14 @@ export type ChangeHandler = (
         plot?: any;
         // Interactive Graph callback (see legacy: interactive-graph.tsx)
         graph?: PerseusGraphType;
+        // widgets/number-line.tsx
+        divisionRange?: number[];
+        // widgets/number-line.ts
+        numDivisions?: number;
+        // widgets/number-line.ts
+        numLinePosition?: number;
+        // widgets/number-line.ts
+        rel?: Relationship;
```

Comment:
Not a result of your PR, but this is really gross. I get the sense that `ChangeHandler` was supposed to take a PerseusItem or PerseusRenderer or something and then other things were just thrown in...

Reply:
At the start, it seemed like change props across our widgets used the same type signature. In reality, each widget's `onChange` prop should be defined as the set of options that the widget handles (and I think in some cases it's even more custom than that). 

In reality, the `Renderer` _does_ pass down the same `onChange` value to all widgets, so they do share the same type, and we've gotten away with it because [in the Renderer](https://github.com/Khan/perseus/blob/4ac22bdadcdad844ed9964485ef727de53c6dfa7/packages/perseus/src/renderer.tsx#L602) there's no typing on the function (lots of `any` in this area!).